civihr_employee_portal/views/templates/views-view-table--Tasks--block.tpl.php

```diff
@@ -186,17 +176,13 @@ class="ctools-use-modal ctools-modal-civihr-custom-style ctools-use-modal-proces
     (function($){
```

Comment:
Why we are adding javascript code in tpl.php file , this should go into some .js file.